reference/variables.md

```diff
@@ -289,6 +289,12 @@ Example:
 </upstream>
 ```
 
+To check, for example, if the status code of the response with `id="myRequest"` is successful you can use the following XPath expression:
```

Comment:
This looks like there is an `id` attribute with value `myRequest`. In fact, there might be a `content` attribute of the `request` element or an `id` property in the JSON of the `request` element body.

Reply:
-> with the ID `myRequest`